### ISSUE TITLE
Encryption at rest for local backend

### ISSUE THREAD

**Issue by Judekeyser** [Judekeyser/family-calendar#22]
We want to experiment encryption at rest, using Crypto API, for event stored locally. This would simplify the work when running off-line PWA.

**Comment by Judekeyser** [comment]
We are going to close this issue, as it is not yet relevant.

We investigated encryption at rest on the backend. As it turned out, it is viable to perform a slow migration of data, relying on a temporary column that decides whether or not the raw binary is encrypted.

For frontend encryption, it would be sufficient to encrypt the description part of the message. We isolated the usage of the current Map, so that usages are easy to track for later migration. For now, such protection is void, because we do not use IndexedDb yet
